crates/sdk/src/env/ext.rs

```diff
@@ -111,6 +114,13 @@ impl DraftProposal {
 
         ProposalId(buf)
     }
+
+    /// Finalise the proposal and send it to the blockchain.
+    #[must_use]
+    pub fn approve(self, proposal_id: ProposalId) -> ProposalId {
+        unsafe { sys::approve_proposal(BufferMut::new(&proposal_id)) }
+        proposal_id
+    }
```

Comment:
`DraftProposal::approve` is implied on `send`, no need for this here

Reply:
Removed